### PR TITLE
Begin making pyscript.py into a Python package

### PR DESCRIPTION
**Pull request by hoodmane**
This doesn't quite do the full job because we also need to make sure that the rest of the files in the new pyscript folder also get included. Will do this in a followup.

### CODE REVIEW

File: pyscriptjs/src/main.ts [hunk 1]
```diff
@@ -213,7 +213,8 @@ export class PyScriptApp {
         logger.info('importing pyscript');
 
         // Save and load pyscript.py from FS
-        interpreter.interface.FS.writeFile('pyscript.py', pyscript, { encoding: 'utf8' });
```
Comment (hoodmane) [inline pyscriptjs/src/main.ts:216]: `writeFile` does not take an option called `encoding`.

**Review by FabioRosado — APPROVED** [review]

**Comment by marimeireles** [comment]
We're getting a bunch of test failing and shouldn't approve this one yet :) I think Hood is still working on it

**Comment by hoodmane** [comment]
Okay now it's passing. Please merge @FabioRosado. I will open a followup which uses rollup to inject all files in the pyscript directory (currently only the `__init__.py` file is included).

**Comment by FabioRosado** [comment]
Sounds good merged, on the follow up can you add an entry in the changelog.md in the docs folder? 😄

**Comment by hoodmane** [comment]
Okay, will do. Thanks!

**Pull request merged**